### DISCUSSION TITLE
Black line in-between pics

### DISCUSSION THREAD

**Discussion by JustSilent88** [mienaiyami/yomikiru#355] [Q&A] [answered]
Hi is there a way to make the pics joined and not cut by this black line ?
![Screenshot 2024-02-16 225446](https://github.com/mienaiyami/yomikiru/assets/142888207/f8f112b2-2724-4dbe-a57d-54620d5f3a04)

**Comment by mienaiyami** [comment, marked as answer]
You can join them by unchecking "Gap between rows" under "Reader settings (menu button on top right of screen) --> Other Settings (last section).
![image](https://github.com/mienaiyami/yomikiru/assets/84740082/68eab267-04e3-41d5-a7fa-39d44af7aee9)

**Reply by JustSilent88** [reply]
Thank you so much for the fast reply
